contentcuration/contentcuration/frontend/channelEdit/components/edit/EditView.vue

```diff
@@ -243,6 +243,59 @@
       trackTab(name) {
         this.$analytics.trackClick('channel_editor_modal', name);
       },
+      handleErrorClick(error) {
+        const errorRefs = {
+          Title: 'title',
+          License: 'license',
+          'Copyright holder': 'copyright_holder',
+          Completion: 'randomize',
+          'Learning Activity': 'learning_activities',
+        };
+
+        const errorRef = errorRefs[error];
+
+        const targetElement = this.$refs.detailsTab.$refs[errorRef];
+
+        console.log(targetElement, 'target');
+
+        if (!targetElement) {
+          console.error(`Target element ref not found for error: ${error}`);
```

Comment:
we need to add a return statement here to stop the function excecution. Otherwise it will fall in the next statement `const nativeElement = targetElement.$el`.